tidb-cloud/monitor-alert-zoom.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Subscribe via Zoom
+summary: Learn how to monitor your TiDB cluster by getting alert notification via Zoom.
+---
+
+# Subscribe via Zoom
+
+TiDB Cloud provides you with an easy way to subscribe to alert notifications via multiple channels. This document describes how to subscribe to alert notifications via Zoom.
+
+## Prerequisites
+
+- To subscribe to alert notification of TiDB Cloud, you must have the `Organization Owner` access to your organization or `Project Owner` access to the target project in TiDB Cloud.
+
+- To receive alert notifications via Zoom, you need to have an Enterprise support plan or higher.
+
+- To add and configure the Incoming Webhook Chatbot in Zoom, you need to have admin permissions on your Zoom account.
+
+## Subscribe to alert notifications
+
+### Step 1. Add Zoom Incoming Webhook App
+
+1. Sign in to the [Zoom App Marketplace](https://marketplace.zoom.us/) as the account administrator.
+2. Add [Incoming Webhook App](https://marketplace.zoom.us/apps/eH_dLuquRd-VYcOsNGy-hQ) from the Zoom App Marketplace, Click **Add**. If the app is not pre-approved, contact your Zoom admin to approve this app for your account. Learn more about [admin app approvals](https://support.zoom.us/hc/en-us/articles/360027829671). 
+3. Confirm the permissions the app requires, then click **Authorize**. The Incoming Webhook app is now added.
+
+### Step 2. Generate a Zoom webhook URL
+
+1. Sign in to the Zoom desktop client.
+2. Click the **Team Chat** tab.
+3. Under **Apps**, find and select **Incoming Webhook**, or select a chat channel from above that you would like to receive messages in.
+4. Enter the following command to make a new connection, please replace `<connectionName>` with your connection name that you want to use, for example, `tidbcloud-alerts`:
+    ```shell
+    /inc connect <connectionName>
+    ```
+5. The command will return the following details:
+   - **Endpoint**, this will provide a webhook URL in the format: `https://integrations.zoom.us/chat/webhooks/incomingwebhook/XXXXXXXXXXXXXXXXXXXXXXXX`.
```

Comment:
```suggestion
   - **Endpoint**. It will provide a webhook URL in the format: `https://integrations.zoom.us/chat/webhooks/incomingwebhook/XXXXXXXXXXXXXXXXXXXXXXXX`.
```